src/stores/adminOrganisation.ts

```diff
@@ -217,6 +218,33 @@ export const useAdminOrganisationStore = defineStore('adminOrganisation', {
       }
       this.isLoadingApprovePayments = false;
     },
+    /**
+     * Create invoice from internal form state
+     * @returns {Promise<boolean>} - Success status
+     */
+    async createInvoice(): Promise<boolean> {
+      const { makeInvoice } = useApiPostCoordinatorMakeInvoice(this.$log);
+
+      // send all selected member ids
+      const paymentIds: number[] = [];
```

Comment:
We need send payment ID field value (from the REST API GET URL rest/coordinator/invoices/) instead of user profile ID.

Example of JSON output data (see `"id": 178` field value):

```json
{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "payments_to_invoice": [
                {
                    "id": 178,
                    "amount": 119,
                    "userprofile_id": 36,
                    "payment_status": 1005,
                    "pay_type": "fc",
                    "pay_category": "entry_fee"
                }
            ],
            "invoices": []
        }
    ]
}
```